doc.yml

```diff
@@ -0,0 +1,12 @@
+doc:
```

Comment:
I would name the template and job more explicit .. Maybe `confluence-sync`, `confluence-updater` or something else.
What do you think?

Reply:
agreed! renamed :)